pages/common/clj.md

```diff
@@ -0,0 +1,29 @@
+# clj
+
+> Clojure tool to start a REPL or invoke a specific function with data.
+> All options can be defined in a `deps.edn` file.
+> More information: <https://clojure.org/guides/deps_and_cli>.
+
+- Start a REPL:
+
+`clj`
+
+- Exec function:
+
+`clj -X {{namespace/fn_name}}`
+
+- Run -main function of a namespace:
+
+`clj -M -m {{namespace}} {{args}}`
+
+- Prepare (resolve deps, download libraries, make and cache classpaths):
+
+`clj -P`
+
+- Start nREPL server with CIDER middleware:
+
+`clj -Sdeps '{:deps {nrepl {:mvn/version "0.7.0"} cider/cider-nrepl {:mvn/version "0.25.2"}}}' -m nrepl.cmdline --middleware '["cider.nrepl/cider-middleware"]' --interactive`
```

Comment:
What's the difference between this example and the earlier REPL one? This example looks rather long and complicated. Is there a way we could shorten it at all?

Reply:
This one enables external tools to communicate with the REPL through the nREPL API. The way to make it shorter would be removing CIDER middleware which is nice to have but totally optional for most IDEs/tools.
```suggestion
- Start nREPL server to be used with IDEs/tools:

`clj -Sdeps '{:deps {nrepl/nrepl {:mvn/version "0.7.0"}}}' -m nrepl.cmdline --interactive`
```

Reply:
Ah, I see. Thanks for the explanation! Let's keep it as-is then.